File: Cpp/calculator.cpp
#include<iostream>
using namespace std;
int main()
{
	int a,var1,var2;
	cout<<"Enter 1 for addition"<<endl<<"Enter 2 for substraction"<<endl<<"Enter 3 for multiplication"<<endl<<"Enter 4 for divsion"<<endl;
	cin>>a;
	if(a==1)
	{
		cout<<"Enter 2 variables to add"<<endl;
		cin>>var1>>var2;
		cout<<"Addition of given numbers is "<<var1+var2;
	}
	if(a==2)
	{
		cout<<"Enter 2 variables to substract"<<endl;
		cin>>var1>>var2;
		cout<<"substraction of given numbers is "<<var1-var2;
	}
	if(a==3)
	{
		cout<<"Enter 2 variables to multiply"<<endl;
		cin>>var1>>var2;
		cout<<"multiplication of given numbers is "<<var1*var2;
	}
	if(a==4)
	{
		cout<<"Enter 2 variables to divide"<<endl;
		cin>>var1>>var2;
		if(var2==0)
		{
			cout<<"Not difine";
		}
		else
		{	
			cout<<"division of given numbers is "<<var1/var2;
		}
	}
	return 0;
}

File: Python/Calculator.py
#Calculator
count=1
while(count==1):


   a=int(input("1.addition\n2.substraction\n3.multiplication\n4.division\n"))
   if(a==1):
       num1=int(input("Enter variable 1 to add"))
       num2=int(input("Enter variables 2 to add"))
       print("Sum is ",num1+num2)
       count=0
   elif(a==2):
       num1=int(input("Enter variable 1 to substraction"))
       num2=int(input("Enter variables 2 to substraction"))
       print("Sub is ",num1-num2)
       count=0
   elif(a==3):
       num1=int(input("Enter variable 1 to multiply"))
       num2=int(input("Enter variables 2 to multiply"))
       print("multip is ",num1*num2)
       count=0
   elif(a==4):
       num1=int(input("Enter variable 1 to divide"))
       num2=int(input("Enter variables 2 to divide"))
       print("division is ",num1/num2)
       count=0
   else:
       print("\nEnter a valid option\n")
       count=1
